### DISCUSSION TITLE
Website widget icon location

### DISCUSSION THREAD

**Discussion by gonzalo** [chatwoot/chatwoot#5028] [Chatwoot APIs] [answered]
There's anyway to change the default location (down right) for the website widget icon?

In our scenario the widget is inserted inside our tool through an iframe help website. So the icon is not visible without scrolling. We would like to move it to top-right of the iframe window

**Comment by gonzalo** [comment, marked as answer]
Sorry, I just found the answer in this forum https://github.com/chatwoot/chatwoot/discussions/2145

Please remove this message if you consider

**Comment by gonzalo** [comment]
But... (there's always a "but")

  position: 'left', // This can be left or right

The position parameter only allows to select left or right, there's any other parameter that allows me to select "top˝"or "down" ?

**Reply by sojan-official** [reply]
At the moment, only `left,` and `right` is supported. you can look into using a custom element to trigger the bubble instead of the default chat bubble 
https://www.chatwoot.com/docs/product/channels/live-chat/sdk/setup/#to-trigger-widget-without-displaying-bubble

**Comment by gonzalo** [comment]
So far from JS knowledge, maybe one of our developers can help me. But anyway, thanks for your superfast answer. 

BTW We're testing this tool and seems a perfect solution for our needs.
